src/dmd/glue.d

```diff
@@ -1163,8 +1163,67 @@ void FuncDeclaration_toObjFile(FuncDeclaration fd, bool multiobj)
 
     if (fd.isCrtCtorDtor & 1)
         objmod.setModuleCtorDtor(s, true);
+
     if (fd.isCrtCtorDtor & 2)
-        objmod.setModuleCtorDtor(s, false);
+    {
+        bool mitigation = true;
+        /*
+            https://issues.dlang.org/show_bug.cgi?id=22520
+
+            Apple radar: https://openradar.appspot.com/FB9733712
+
+            Apple deprecated the mechanism used to implement `crt_constructor`
```

Comment:
```suggestion
            Apple deprecated the mechanism used to implement `crt_destructor`
```

Reply:
Oops!

Reply:
@maxhaton are you going to fix this comment?